packages/server/src/schema/metadata/resolvers.ts

```diff
@@ -19,6 +19,7 @@
  * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://github.com/serlo-org/api.serlo.org for the canonical source repository
  */
+import { version } from '../../../package.json'
```

Comment:
```suggestion
import { version } from '../../../package.json' as { version: string }
```
or something similar to fix the eslint error.